### DISCUSSION TITLE
curl: (7) Failed to connect to www.nasm.us port 443: Network is unreachable

### DISCUSSION THREAD

**Discussion by snawar92** [ImageMagick/ImageMagick#6498] [Help] [answered]
### ImageMagick version

6

### Operating system

MacOS

### Operating system, version and so on

10.13.6

### Description

after brew install facing this issue 

==> Fetching nasm
Error: imagemagick@6: Failed to download resource "nasm"
Failure while executing; `/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.0.29\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 10.13.6\)\ curl/7.54.0 --header Accept-Language:\ en --retry 3 --fail --location --silent --head https://www.nasm.us/pub/nasm/releasebuilds/2.16.01/nasm-2.16.01.tar.xz` exited with 7. Here's the output:
curl: (7) Failed to connect to www.nasm.us port 443: Network is unreachable

### Steps to Reproduce

brew install imagemagick@6

**Comment by fmw42** [comment]
Please don't post images of code or errors! Post the actual text.

**Comment by snawar92** [comment]
> Please don't post images of code or errors!

while posting issue there was mandatory option ,  any thoughts on above issue @fmw42 ?

**Comment by fmw42** [comment]
The error message says "Network is unreachable".  That means either your curl failed on your end or on the Homebrew end.  This has nothing to do with installing Imagemagick.

**Comment by fmw42** [comment]
NASM.us

NASM
https://www.nasm.us
This is the project webpage for the Netwide Assembler (NASM), an asssembler for the x86 CPU architecture portable to nearly every modern platform, and with code ...

Netwide Assembler does not seem to me to relevant to Homebrew.  But I do not use Homebrew.

Are you sure you copied the curl command to download Homebrew correctly?

**Comment by fmw42** [comment]
Instructions for installing from Homebrew can be found at https://imagemagick.org/script/download.php#macosx

**Reply by snawar92** [reply]
is there any way we can skip this dependency --> NASM ?

**Comment by snawar92** [comment, marked as answer]
installing ImageMagick from homebrew is creating issues now in old macs, by using macports it worked for me @fmw42
